view/next-project/src/pages/sponsors/index.tsx

```diff
@@ -1,42 +1,57 @@
 import clsx from 'clsx';
 import type { NextPage } from 'next';
 import Head from 'next/head';
-import { useState as UseState, useMemo as UseMemo } from 'react';
-
+import { useState as UseState, useEffect as UseEffect } from 'react';
```

Comment:
```suggestion
import { useState, useEffect  } from 'react';
```
別名で定義する必要はないので修正してください。
これに伴うコードの修正もお願いします。